ext/net/ops.rs

```diff
@@ -180,14 +182,27 @@ pub async fn op_net_accept_tcp(
     .try_or_cancel(cancel)
     .await
     .map_err(accept_err)?;
+  let mut _fd_raw: Option<Fd> = None;
```

Comment:
nit: probably this shouldn't be prefixed with `_` as the value is used later.

Reply:
@kt3k , yes I initially had `fd_raw`, but the lint on linux complained about
```
error: value assigned to `fd_raw` is never read
   --> ext/net/ops.rs:185:11
    |
185 |   let mut fd_raw: Option<Fd> = None;
    |           ^^^^^^
    |
    = help: maybe it is overwritten before being read?
    = note: `-D unused-assignments` implied by `-D warnings`
    = help: to override `-D warnings` add `#[allow(unused_assignments)]`


```

Reply:
Ok. Makes sense